lib/transform.ml

```diff
@@ -1,5 +1,7 @@
 open Shape
 
+type transform = (shape -> shape)
```

Comment:
This is already in the mli file. Does it need to be here?

Reply:
I don't know, can a type just be declared in the .mli file and not in the .ml file? I had assumed it had to be present in both.

Reply:
It doesn't compile unless its provided in both.